makefiles/toolchain/gnu.inc.mk

```diff
@@ -22,5 +22,13 @@ endif
 # Default to the native (g)objdump, helps when using toolchain from docker
 _OBJDUMP         := $(or $(shell command -v $(PREFIX)objdump || command -v gobjdump),objdump)
 OBJDUMP   ?= $(_OBJDUMP)
+
+GCC_VERSION := $(shell $(CC) -dumpversion)
+
+# -fmacro-prefix-map requires GCC 8
+ifneq (8, $(firstword $(shell echo "8\n$(GCC_VERSION)" | sort -V))))
```

Comment:
This is not portable, as replacing `\n` with a newline character is not POSIX standard. Bash will do so, but not e.g. `ash`. Also, `-V` seems not to not work with all implementations of `sort`.

Reply:
I hope it's portable now